src/PdfSharper/Pdf.AcroForms/PdfSignatureField.cs

```diff
@@ -112,7 +110,6 @@ public PdfSignatureField(PdfDocument document)
             sign.Elements.Add(Keys.M, new PdfDate(DateTime.Now));
 
             document._irefTable.Add(sign);
-            document._irefTable.Add(this);
```

Comment:
Do we know where this is getting added at? Our code or internally to pdf sharper?